### DISCUSSION TITLE
How to change S3 endpoint address

### DISCUSSION THREAD

**Discussion by dkoci** [noobaa/noobaa-core#6834] [Q&A]
Hi.
I have deployed noobaa using noobaa-operator. I dont want to use default routes. I have created my own routes but I dont know how to change S3 endpoint address (address is still the old one which I would like to replace).

![image](https://user-images.githubusercontent.com/5565897/146574133-7db3158c-e701-4c71-9c8b-d7d0cad49156.png)

**Comment by jackyalbo** [comment]
@dkoci Can you share how your route is configured? I mean without any private info... Did you delete the old s3 route? Is the new one configure with noobaa's s3 service?

**Reply by dkoci** [reply]
I created new route with s3 service. It is impossible to delete the old endpoint route because it is automatically recreated as soon I delete it. I would like to make my new endpodint route visible in mgmt gui.
I want to change the s3-ednpoint route only (mgmt route remains the same).

**Reply by jackyalbo** [reply]
Can you just edit the route? Or create it with your own details? I didn't try it by myself. 
You can maybe turn off the noobaa-operator (set replicas to 0 on the deployment) while editing or creating a new route with your new setup under the s3 name.

**Reply by dkoci** [reply]
I turned off noobaa-operator, deleted s3 route and created a new one. There is still the old s3 endpoint route visible in mgmt gui (no new one).

**Reply by jackyalbo** [reply]
ok, so two things that I want to check: Can you reach the endpoint? Does S3 calls work for you after this change?
If the issue is only on the UI. I think you can try to delete the core pod and see if that does the trick.

**Reply by dkoci** [reply]
Core pod termination did the trick. It is the new s3-endpoint available in mgmt gui. Thank you.
Can you help me with another thing :) : https://github.com/noobaa/noobaa-core/discussions/6835
